jenkins-pipelines/perf-regression-alternator-throughput-30min-ttl.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
```

Comment:
Please remove this test (code, config and pipeline) because it doesn't really test it well.

Reply:
fixed